### DISCUSSION TITLE
Support for Claude?

### DISCUSSION THREAD

**Discussion by brando90** [stanfordnlp/dspy#1264] [Q&A]
Is there an example of how to use Claude with DSPy? I'm happy to help build it if needed. Or any pointers would be nice. 

Thanks in advance!

**Comment by d2kagw** [comment]
Hey @brando90,
Here's an example I posted earlier:
https://github.com/stanfordnlp/dspy/discussions/1261

(although I am having some issues getting it running 🤣)

**Reply by brando90** [reply]
found this too https://dspy-docs.vercel.app/api/language_model_clients/aws_models will see options later.

**Reply by brando90** [reply]
https://gist.github.com/yshaaban/274e70141fd0f54fd3a88625d46f3a1c
